mobile-app/src/domain/alert/screens/alert.screen.tsx

```diff
@@ -29,33 +29,57 @@ const AlertScreen = () => {
 		location: {} as DeviceLocation,
 	});
 
+	const [previousAppState, setPreviousAppState] =
+		useState<AppStateStatus>('unknown');
+
 	const { fullName } = useUserInfoStore();
 	const { contacts } = useContactStore();
 
 	const [minutes, setMinutes] = useState(0);
 	const [hint, setHint] = useState<string>();
 
 	const {
-		permissionsGranted: locationPermissionsGranted,
+		permissionGranted: locationPermissionsGranted,
 		getHighPriorityLocation,
 		getLowPriorityLocation,
 	} = useLocation();
 
+	const updateLowPriorityLocationContext = () => {
+		getLowPriorityLocation().then(location => {
+			setContext((current) => {
+				return { ...current, location };
+			});
+		});
+	}
+
+	useEffect(() => {
+		updateLowPriorityLocationContext();
+	}, []);
```

Comment:
Iako ne vidim grešku ovde, najbolja praksa je da uvek u dependency array uključiš sve što se koristi unutar efekta.
Baci pogled na ovo: https://react.dev/learn/removing-effect-dependencies

To znači da bi i `updateLowPriorityLocationContext` morao wrappovati sa use callback i ispoštovati njegov deps.
Ili možeš preskočiti `updateLowPriorityLocationContext` skroz i `getLowPriorityLocation` zvati u useEffectu direktno, a možeš i korak dalje pa sve to wrappovati u custom hook tipa 
`const { lowPriorityLocation } = useLowPriorityLocation()` 🤣